generator/js_generator.cc

```diff
@@ -2237,7 +2237,7 @@ void Generator::GenerateClassFieldInfo(const GeneratorOptions& options,
       if (IgnoreOneof(desc->oneof_decl(i))) {
         continue;
       }
-      GenerateOneofCaseDefinition(options, printer, desc->oneof_decl(i));
+      GenerateOneofCaseDefinition(options, printer, desc->oneof_decl(i), desc);
```

Comment:
There should be no need to pass desc since it can be obtained from the OneofDescriptor.

Reply:
done